mapo/agents/mapo/mapo_policy.py

```diff
@@ -10,23 +12,67 @@
 from ray.rllib.evaluation.postprocessing import compute_advantages, Postprocessing
 
 
-def build_actor_critic_losses(policy, batch_tensors):
-    """Contruct actor (DPG) and critic (Fitted Q) tf losses."""
+AgentComponents = namedtuple("AgentComponents", "dynamics critic actor")
```

Comment:
What about `AgentModel` ?

Reply:
I just wanted something to name tuple components by the models they appertain to. I couldn't think of a better name.